examples/model_selection/plot_precision_recall.py

```diff
@@ -127,6 +127,21 @@
                                                      average="micro")
 
 
+# Plot ISO-F1 curve
```

Comment:
ISO is international standards organisation. ["Iso-"](https://en.wiktionary.org/wiki/iso-) is a greek-derived prefix meaning equal